en/components/for_of.md

```diff
@@ -0,0 +1,149 @@
+---
+title: igxFor
+_description: The Ignite UI for Angular product is providing virtual directive igxFor similar to ngFor, but it is creating scrollbars and only renders small chunks of the data.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Virtualization, Performance, Virtual directive, Virtual For
+---
+
+## igxFor
+
+<p class="highlight">Display huge amount of repeated items. The directive is creating scrollable containers and renders small chunks of the data. Is is used inside igx-grid and it can be used to build virtual igx-list.</p>
```

Comment:
Virtualizes DOM object rendering by visualizing only the visible chunks of the data in the DOM. DOM objects are reused while the end-user is scrolling. The igx-grid implements the igxFor directive internally, but the directive can also be implemented on any component that needs to render large amounts of data, like the igx-list.